Assets/_BForBoss/_UserInterface/Scripts/UserInterface.asmdef

```diff
@@ -3,7 +3,8 @@
     "rootNamespace": "Perigon.UserInterface",
     "references": [
         "GUID:6055be8ebefd69e48b49212b09b47b2f",
-        "GUID:b415fcadef204ad4e81c959b085bc506"
+        "GUID:b415fcadef204ad4e81c959b085bc506",
+        "GUID:0c752da273b17c547ae705acf0f2adf2"
```

Comment:
Why does this need FMOD?

Reply:
Oh good catch, I moved things around and forgot to clean this up